src/main/resources/db/changelog/changes/create-accommodation-amenities-table.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
```

Comment:
If I not mistaken, this changelog should be deleted.

Reply:
Oh, you are right ) this comes from previous changes